net-im/swift/swift-4.0.2.ebuild

```diff
@@ -0,0 +1,175 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit gnome2-utils scons-utils toolchain-funcs
+
+DESCRIPTION="An elegant, secure, adaptable and intuitive XMPP Client"
+HOMEPAGE="https://www.swift.im/"
+SRC_URI="https://swift.im/downloads/releases/${P}/${P}.tar.gz"
+
+LICENSE="BSD BSD-1 CC-BY-3.0 GPL-3 OFL-1.1"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+IUSE="client expat examples experimental ft gconf hunspell lua icu idn test tools unbound zeroconf"
```

Comment:
If you need to fix something else, you could maybe sort this & RDEPEND list for that same commit.

Reply:
Good catch, thanks.